TurkishFilter/sections/specific.txt

```diff
@@ -128,6 +128,7 @@ geyvemedya.com##div[id^="reklam-"]
 pchocasi.com.tr###secondary > .sidebar300
 pchocasi.com.tr##a[href*="//bit.ly"][target="_blank"] img
 pchocasi.com.tr##a[class^="sidebar-reklam-"]
+||kimintelefonu.com/*_reklam
```

Comment:
You may want to hide the ad divider too:

clear:both;height:30px

Reply:
it is redundant